articles/security/token-exp.md

```diff
@@ -0,0 +1,41 @@
+# Token Expiration 
+
+This document states the expiration of different types of tokens that are referenced in Auth0 documentation. [Click here to learn more about the types of tokens.](/tokens)
+
+## Auth0 `access_token`
```

Comment:
If this refers to the Management APIv1 Access Token then let's make the explicit mention to it. `Auth0 access_token` is misleading as a title.